app/Http/Controllers/RecruitingCampaignRecipientController.php

```diff
@@ -27,7 +27,48 @@ public function index()
      */
     public function store(Request $request)
     {
-        return response()->json(['status' => 'error', 'error' => 'not_implemented'], 501);
+        $this->validate($request, [
+            'recruiting_campaign_id' => 'exists:recruiting_campaigns,id|numeric',
+            'recipients' => 'required_unless:addresses|array',
+            'address' => 'required_unless:recipients',
+            'recruiting_visit_id' => 'exists:recruiting_visits,id',
+            'user_id' => 'exists:users,id|numeric',
+        ]);
+
+        if ($request->filled('recipients')) {
+            $recipients = $request->input('recipients');
+        } else {
+            $recipients[0]['address'] = $request->input('address');
+            $recipients[0]['recruiting_visit_id'] = $request->input('address');
```

Comment:
Typo?

Reply:
Also should you be setting `recruiting_campaign_id` for this recipient?

Reply:
>Typo?
Yes

>Also should you be setting recruiting_campaign_id for this recipient?
Assumption was one campaign per request